x-pack/plugins/cloud_security_posture/common/constants.ts

```diff
@@ -40,18 +40,18 @@ export const DETECTION_RULE_ALERTS_STATUS_API_CURRENT_VERSION = '1';
 export const DETECTION_RULE_RULES_API_CURRENT_VERSION = '2023-10-31';
 
 export const CLOUD_SECURITY_POSTURE_PACKAGE_NAME = 'cloud_security_posture';
-// TODO: REMOVE CSP_LATEST_FINDINGS_DATA_VIEW and replace it with LATEST_FINDINGS_INDEX_PATTERN
-export const CSP_LATEST_FINDINGS_DATA_VIEW = 'logs-cloud_security_posture.findings_latest-*';
 
 export const FINDINGS_INDEX_NAME = 'logs-cloud_security_posture.findings';
 export const FINDINGS_INDEX_PATTERN = 'logs-cloud_security_posture.findings-default*';
 export const FINDINGS_INDEX_DEFAULT_NS = 'logs-cloud_security_posture.findings-default';
 
 export const LATEST_FINDINGS_INDEX_TEMPLATE_NAME = 'logs-cloud_security_posture.findings_latest';
+// Defined in https://github.com/elastic/integrations/tree/main/packages/cloud_security_posture/kibana/index_pattern
```

Comment:
are we sure IDs do not change over time due to some reason? to me, this coupling looks a bit brittle, but I'm also don't know much about the IDs

Reply:
They are static once generated for the first time. The index pattern can be updated manually to add more fields but the id will remain the same, only reason for us to change is if we decide to maintain a more "friendly" id like the one that's used by the security solution index (`security-solution-default`), but that would be a breaking change and could lead to issues with migration.

I think using the id is better than searching the name for this server side function as the name could be error prone if for some reason the user create another dataview with the same name

Reply:
thanks for the context, I think it makes sense